particles/Restaurants/Reservations.recipes

```diff
@@ -7,59 +7,87 @@
 
 import '../List/List.recipes'
 import '../Layout/Layout.recipes'
+import '../Events/Events.recipes'
 import '../Restaurants/Restaurant.schema'
-import '../Events/Event.schema'
 import '../Arcs/Description.schema'
 
+// particle RestaurantEvent in '../Common/source/Noop.js'
+//   in Event event
+//   in [Restaurant] restaurants
+//   consume root
+
+// // TODO(sjmiles): doesn't coalesce like this, needs more ... something (slots? handle-connections?)
+// recipe ReservationEvent
+//   create #reservation as event
+//   use as restaurants
+//   RestaurantEvent
+//     event = event
+//     restaurants = restaurants
+//     //consume toproot as toproot
+
+recipe ReservationCalendar
+  //use #reservation as event
```

Comment:
just curios - all the various commented out parts of recipes - why do you want to keep them? 
is it something that you tried, but didn't work, but should work?
maybe worth adding little notes about them?

Reply:
I tend to leave commented-code around for one generation so I can make sure I've extracted all the value. I'll try to do it less, I understand it can look sloppy.